src/telegram.ts

```diff
@@ -269,7 +269,7 @@ class Telegram extends ApiClient {
    */
   sendSticker(
     chatId: number | string,
-    sticker: tt.InputFile,
+    sticker: tt.Opts<'sendSticker'>['sticker'],
```

Comment:
It is important to note here that `string` must not be part of `InputFile`. It is not possible to use a URL or a file_id when sending a sticker, and `typegram` already models this correctly. The correct type is simply `InputFile` as it is defined.

Reply:
Also confer https://github.com/KnorpelSenf/typegram/issues/15